tests/plugins/nvim-lsp.nix

```diff
@@ -0,0 +1,40 @@
+{
+  empty = {
+    plugins.lsp.enable = true;
+  };
+
+  test = {
+    plugins.lsp = {
+      enable = true;
+
+      keymaps = {
+        silent = true;
+        diagnostic = {
+          "<leader>k" = "goto_prev";
+          "<leader>j" = "goto_next";
+        };
+
+        lspBuf = {
+          "gd" = "definition";
+          "gD" = "references";
+          "gt" = "type_definition";
+          "gi" = "implementation";
+          "K" = "hover";
+        };
+      };
+
+      servers = {
+        bashls.enable = true;
+        clangd.enable = true;
+        nil_ls.enable = true;
+        rust-analyzer.enable = true;
+        pylsp = {
+          enable = true;
+          # TODO uncomment
+          # filetypes = ["python"];
+          # autostart = false;
```

Comment:
TODO in the code :P I approved the other PR so you should be able to change it

Reply:
done